src/types.ts

```diff
@@ -31,6 +32,19 @@ export const asEdgeBox = asObject({
 export type StoreFileTimestampMap = ReturnType<typeof asStoreFileTimestampMap>
 export const asStoreFileTimestampMap = asMap(asNumber)
 
+// Settings Document
+
+export type StoreSettings = ReturnType<typeof asStoreSettings>
+export type StoreSettingsDocument = ReturnType<typeof asStoreSettingsDocument>
+export const asStoreSettings = asObject({
+  ipWhitelist: asArray(asString),
```

Comment:
The white list and API key list should be maps instead of arrays

Reply:
I'll change to `asMap(asBoolean)`.